_data/sites.json

```diff
@@ -8093,6 +8093,18 @@
         ]
     },
 
+    {
+        "name": "Pokémon GO",
+        "url": "https://niantic.helpshift.com/a/pokemon-go/?s=account-sign-in&f=how-do-i-delete-my-pokemon-go-account&l=en&p=web",
+        "difficulty": "easy",
```

Comment:
I think it should be changed to hard, even though the steps in itself are relatively straight forward. If it takes weeks to process, it seems like it requires human interaction, and that would mean it is not automatic. Easy are typically the ones where your account is purged immediately.